wsf_scraping/settings.py

```diff
@@ -80,10 +80,13 @@
 FEED_TEMPDIR = 'var/tmp/'
 
 if FEED_CONFIG == 'AWS':
-    AWS_S3_BUCKET = 'data-labs'
-    AWS_S3_FILE_NAME = 'scraper-results/%(name)s - %(time)s.json'
+    AWS_S3_BUCKET = 'data-labs-data'
+    AWS_S3_FILE_NAME = 'scraper-results/%(name)s'
     DATABASE_ADAPTOR = 'dynamodb'
-    FEED_URI = f'aws://{AWS_S3_BUCKET}/{AWS_S3_FILE_NAME}'
```

Comment:
not a fan of the new string type? it seemed cool when i read about i :p

Reply:
Didn't want to depend too much on a python version, just in case, but I like the new string type 😄